src/components/Authors/Authors.tsx

```diff
@@ -0,0 +1,106 @@
+
+import React, { useEffect, useState } from 'react';
+import { format } from 'date-fns';
+import styled from '@emotion/styled';
+import axios from 'axios';
+
+/*
+Require: config.features.editOnRepo.editable = true
+We collect `Author` field, ref: https://stackoverflow.com/a/6755848/4540808
+*/
+
+export interface IAuthorsProps{
+  className: string;
+  path: string;
+  repoType: string;
+  repo: string;
+  contentRootPath: string;
+  locationPathname: string;
+}
+
+export interface IAuthorInfo{
+  username?: string; 
+  name: string;
+  commits: number;
+  url: string;
```

Comment:
I suggest to update some properties to more specific meaning,e.g.

```ts

interface IAuthorInfo {
  username?: string;
  commitsCount: number;
  profileUrl: string;
}
```


I think we can remove name and use username for displaying and react list key because it is unique value.

Reply:
I want to show the `name` prop instead of `username` prop.

Reply:
@mildronize  but we can keep only one property right?